pyproject.toml

```diff
@@ -61,7 +61,7 @@ django-mptt = "~0.11.0"
 # Prometheus metrics for Django
 django-prometheus = "~2.1.0"
 # Redis cache for Django used for distributed locking
-django-redis = "~4.12.1"
+django-redis = "~5.1.0"
```

Comment:
You mention in the PR description that the configuration is "drastically different". Do we need information in the 1.2 release-notes about how users might need to update their existing `nautobot_config.py` when migrating to 1.2?

Reply:
the config is the same for the normal redis configuration, but it is very different for sentinel.  I'm not sure how many users use sentinel but it's probably worth a note that configuration for Sentinel has been updated.

Reply:
Would you be willing to take a stab at writing such a note?